data/changelog.txt

```diff
@@ -1,33 +1,461 @@
+Aftershock: Two new heavy laser cannons
+Dreamsmiths got their spells
 
 
 ## Balance:
+Adds assorted costumes to the objective of the Cosplay mission.
+Use find_clear_path always for projectile trajectories, not just when it hits.
+Makes using a spear on enemies through fences/bars normally at low levels instead of defaulting to bashing on bars/fence instead.
+Increases .357 SIG’s overall armour penetration to bring it on par with similar ammo.
+Fixing density and some recipes to match reality better
+Five pin bow sights are uncraftable
+make fewer chain links per craft
+Add additional sources of patchwork cotton sheets from deconstruction
+Mole Crickets and Molerats can now give tetanus with bites
+Number changes for half worms
+Pipe and pipe weapons are no longer durable
+Nerf the Bokken but make the recipe easier
+CBMs are now priced according to their relative utility
+Meat scrap price fixes
+Gives chocolate the stats of a Hershey's bar
+Batch-time savings for cooking plant marrow
+make Phelloderm not awful
+improve chimera mutants' armor
+Revamps the spawns of ferrous/rust zombies
+Some spell adjustments to make them more equal
+Aluminum armor values rebalance
+Revamps the monster spawns in fitness related places
+Rebalanced tool belt weight and volume
+Remove bonus stab damage from BB gun
+Non-lethal scratches
+Make integrated shells outer in addition to strapped
+Rework basement monster spawns
+Add sounds for shockers
+Faraday chainmail is tempered steel, lower encumbrance, doesn't have rubber boots
+RM42 Fighting Knife tweaks
+wilderness spawn rework
+Change military composite armor removal requirements to power tool level metal sawing
+Change shotgun speedloader dimensions/weight and add belt clip property
+most anatomy mutations are 0 points
+Cap melee skill gain based on monster melee skill
+simple deconstruct is much faster
+Adds better monster spawning to the light industry location
+Small audit to the monster spawns of some helicopter related places
+Butchering Rack Recipe Tweak
+Add denim as a material and buff jeans
+Adjustments to ASRGs
+Yet more item cleanup and fixes
+Reduce power costs of ODS bionic
+Don't telefrag the player in most cases
+Removes phantom melee attack that can disrupt aim from explosive/taser hacks.
+Snow Shovel Qualities Fix
+Placeholder activity levels for many recipes replaced with realistic values
+What Zombies should track by scent?
+cody can make chainmail armor and charges more
+Placeholder activity levels for many mod recipes replaced with appropriate values
+greatly reduce the severity of nyctophobia, also removing hallucinations from it
+welding power costs increase
+Removed fire drill refill recipes, added notched fire boards as ammo
+Add environmental protection to rolled "turtleneck" and "turtleneck shirt" -- the same amount as their unrolled variants
+tone down rm13 a little
+Rework melee, unarmed, dodge, cutting, stabbing, and bashing practice recipes.
```

Comment:
```suggestion
Default to bashing on bars/fence instead of the ground
Cap melee skill gain based on monster melee skill
Simple deconstruct is much faster
Add denim as a material and buff jeans
Removed scent tracking from certain zombies
Cody can make chainmail armor and charges more
Rework melee, unarmed, dodge, cutting, stabbing, and bashing practice recipes.
```